src/Driver/driver_configs.jl

```diff
@@ -76,10 +77,12 @@ struct DriverConfiguration{FT}
         numerical_flux_gradient::NumericalFluxGradient,
         config_info::ConfigSpecificInfo,
     )
+        # FIXME: Once variable degree kernels are merged, remove this assert
+        @assert polyorders[1] === polyorders[2]
```

Comment:
I just realized this is probably not a good idea, since `===` means that the two things you're comparing are the _same_ object (i.e. point to the same location in memory, but what we really want is `==` which compares by value. Is this right @jkozdon ?

Reply:
In this case it doesn't matter since you are comparing two integers not two arrays. With arrays, it is true that `===` compares the pointers and `==` compares the elements of the arrays

Reply:
I changed it to `==`. Although it didn't make a difference for this case, I also agree that for integers comparison it's better to use `==`